concordia/static/css/base.css

```diff
@@ -60,7 +60,7 @@ main {
     min-height: var(--height-main);
 }
 
-h1 {
+h1, .h1 {
```

Comment:
I'm not in love with a class matching a tag name since that doesn't explain what fraction of the semantics we want to match. One way to solve this would be to use a class name like `campaign-title`. We could also use multiple `<h1>` tags but that seems to be an area for concern with screen readers which don't follow the HTML5 spec completely and probably isn't worth the trouble of setting the ARIA attributes & testing.

Reply:
I'm not creating anything new as .h1 classname is from Bootstrap: https://getbootstrap.com/docs/4.1/content/typography/#headings. We can definitely create a new classname but I thought our goal was to use bootstrap classes if the style is available. But yes I think we can discuss more about this once we start using sass version.

Reply:
As long as this is supposed to be identical to an h1 in every way except outline level that’s fine.